docs/content_management/pages/react_app_block.md

```diff
@@ -102,7 +105,7 @@ Files in this directory create a map of Components which then are imported to `r
 As a result, the components are rendered on the page. 
 
 ``` js
-import Calculator from "../../../page-builder/components/Calculator";
+import Calculator from "/assets/page-builder/components/Calculator";
```

Comment:
I prefer the relative path. Again, from my point of view, the absolute can lead to a miss interpretation of what the root is.

In vendor/ibexa, I found no use of absolute path, only relative paths or package paths. Including this insane one: https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/public/js/scripts/widgets/flatpickr.js#L1

It was shortened in https://github.com/ezsystems/developer-documentation/pull/2074/files#diff-8f99068e452e1e02045fc9d5b04c5162e7f5ee2380a95990cf64be4742ae587aL105-R106 which is part of the conflict.